assets/stylesheets/contributions.scss

```diff
@@ -876,13 +915,56 @@ $currencies: gbp usd aud eur nzd cad;
         .contributions__heading {
             margin-top: 0;
             padding-top: 12px;
+
+            h1, h2 {
+                font-family: 'GHGuardianHeadline-Bold';
+            }
+
+            @media (max-width: 61.24em) {
```

Comment:
could you use the sass media query mixins for this? they already have variables for the breakpoints

e.g.

```sass
@include mq(tablet) {
  ...
}
```